python/ql/lib/semmle/python/ApiGraphs.qll

```diff
@@ -345,6 +354,9 @@ module API {
       result = callee.getReturn() and
       result.getAnImmediateUse() = this
     }
+
+    /** Gets the number of arguments of this call. Both positional and named arguments are counted. */
```

Comment:
```suggestion
    /**
     * Gets the number of positional arguments of this call.
     *
     * Note: This is used for `WithArity[<n>]` in modeling-as-data, where we thought 
     * including keyword arguments didn't make much sense.
     */
```

I thought it would be nice to give some context on why this predicate is implemented like it is.